core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/ApiServiceSpec.scala

```diff
@@ -200,6 +201,12 @@ trait ApiServiceSpec extends TestDriverComponentWithFlatSpecAndMatchers with Raw
       requesterPaysSetupService
     )_
 
+    override val entityServiceConstructor = EntityService.constructor(
+      slickDataSource,
+      samDAO,
+      workbenchMetricBaseName
+    )_
```

Comment:
wait what? What does the trailing underscore do here? Is this what allows callers to supply the `UserInfo`? I haven't seen this syntax.

Reply:
I'm not entirely sure, I haven't seen it much either, but it would seem so? It's used with the other service constructors in this file so I followed suit

Reply:
this is a little bit of scala trivia: EntityService.constructor is _method_ which are not assignable. The underscore converts the method to a function which is assignable.

https://stackoverflow.com/questions/2529184/difference-between-method-and-function-in-scala